app/helpers/zendesk_service_helper.rb

```diff
@@ -63,7 +63,7 @@ def find_end_user(name, email, phone, exact_match: false)
     end
 
     search_string = ""
-    search_string += "name:\"#{name}\" " if name.present?
+    search_string += "name:\"#{qualify_user_name(name)}\" " if name.present?
```

Comment:
I don't think we should include the "(Fake User)" string in the search. But open to being convinced otherwise.

Reply:
I suppose the underlying question is whether or not Zendesk quoted name searches will find a partial match. If they can, that may introduce other problems ("john smith" vs "john smithfield"). I was assuming that the quotes in line 66 meant we wanted something like an exact match. If that's the case, then having the (Fake User) string present is required to match at all, since it's in the name of the Zendesk user on their end.